client/src/components/dashboard/DashboardEditor.jsx

```diff
@@ -3,13 +3,12 @@ import PropTypes from 'prop-types';
 import ReactGridLayout from 'react-grid-layout';
 import DashboardVisualisationList from './DashboardVisualisationList';
 import DashboardCanvasItem from './DashboardCanvasItem';
+import getArrayFromObject from '../../utilities/general';
```

Comment:
I believe `getArrayFromObject` is the same as `Object.values` (not yet standard, needs to be polyfilled) or Lodash [_.values](https://lodash.com/docs/4.17.4#values).

As we're already using lodash, perhaps use `_.values` instead of our own util?